### PR TITLE
[Fix ] Update docs and tutorials for MvP training and evaluation

### PR DESCRIPTION
**Pull request by wqyin**
1. Rearranged the file structure for training data, testing data, and pretrained weights.
2. Updated the tutorials and added detailed instructions on the `Getting Started` page.
3. Updated the config files for easier starting. Avoided renaming downloaded files.

### CODE REVIEW

File: docs/en/tools/eval_model.md [hunk 1]
```diff
@@ -79,14 +80,15 @@ sh ROOT/scripts/val_mvp.sh ${NUM_GPUS} ${CFG_FILE} ${MODEL_PATH}
 
 Example:
 ```shell
-sh ROOT/scripts/val_mvp.sh 8 configs/mvp/shelf_config/mvp_shelf.py weight/xrmocap_mvp_shelf.pth.tar
+sh ROOT/scripts/val_mvp.sh 8 configs/mvp/shelf_config/mvp_shelf.py weight/mvp/xrmocap_mvp_shelf-22d1b5ed_20220831.pth
```
Comment (jiaqiAA) [inline docs/en/tools/eval_model.md:83]: `eval_mvp.sh`?

**Review by jiaqiAA — APPROVED** [review]

File: configs/mvp/campus_config/mvp_campus.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # finetune with the weight pretrained from panoptic dataset
 # using cam3, cam12, cam23
+# yapf: disable
```
Comment (yl-1993) [inline configs/mvp/campus_config/mvp_campus.py:3]: If `noqa` is used below, we may not need `yapf: disable` anymore?

File: docs/en/getting_started.md [hunk 3]
```diff
@@ -174,65 +178,99 @@ python tools/mview_mperson_evaluation.py \
...
+sh scripts/download_install_deformable.sh
 ```
 
-Example:
-```shell
-sh ./scripts/val_mvp.sh 8 configs/mvp/shelf_config/mvp_shelf.py weight/xrmocap_mvp_shelf.pth.tar
+2. Download dataset and pretrained model, taking Shelf dataset as an example:
+
+```bash
+# download Shelf dataset
```
Comment (yl-1993) [inline docs/en/getting_started.md:192]: What about `# download Shelf dataset (16G)`?
Knowing the dataset size may encourage users to set aside some time for data downloading.

**Review by yl-1993 — APPROVED** [review]

File: docs/en/getting_started.md (hunk 3 above)
Comment (wqyin) [reply docs/en/getting_started.md]: Added in [bb71f8e](https://github.com/openxrlab/xrmocap/pull/29/commits/bb71f8eef16e36d321b7ee98c8689cb11bdda78c).

File: configs/mvp/campus_config/mvp_campus.py (hunk 2 above)
Comment (wqyin) [reply configs/mvp/campus_config/mvp_campus.py]: Fixed in [bb71f8e](https://github.com/openxrlab/xrmocap/pull/29/commits/bb71f8eef16e36d321b7ee98c8689cb11bdda78c).

**Pull request merged**